django_zxcvbn_password_validator/zxcvbn_password_validator.py

```diff
@@ -87,9 +87,9 @@ def append_translated_feedback(old_feedbacks, feedback_type, new_feedbacks):
     def get_help_text(self):
         expectations = _("We expect")
         if self.password_minimal_strength == 0:
-            expectations += f" {_('nothing: you can use any password you want.')}"
+            expectations += " " + _('nothing: you can use any password you want.')
             return expectations
-        expectations += f" {_('a password that cannot be guessed')}"
+        expectations += " " + _('a password that cannot be guessed')
```

Comment:
Why is this part modified ?

Reply:
🤷‍♂️ Don't know why but `makemessages` doesn't collect the translations here

Reply:
🤷‍♂️ Don't know why but makemessages doesn't collect the translations here
So they are suppressed from the .po file. Maybe a recent modification in how gettext work ?